api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -975,3 +1051,11 @@ def test_send_the_invalid_type_page_or_page_size_param_returns_400(
     assert response.status_code == status.HTTP_400_BAD_REQUEST
     response_json = response.json()
     assert response_json == error_response
+
+
+# def test_handle_installation_deleted_with_non_existing_installation(mocker: MockerFixture):
+#     payload = {"installation": {"id": 456}}
+#     handle_installation_deleted(payload)
+#     mock_logger.error.assert_called_once_with(
+#         "Github Configuration with installation_id 456 does not exist"
+#     )
```

Comment:
Extra commented out code. Should be deleted, yes?

Reply:
Oops. Thanks! Deleted